containers/heading/UpgradeButton.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+
+import { Icon } from 'react-components';
+
+const UpgradeButton = ({ className }) => {
+    return (
+        <Link to="/settings/subscription" className={className}>
```

Comment:
Just check if `Link` works on apps that doesn't support this route please.

Reply:
it doesn't